src/ipc/handler.c

```diff
@@ -1302,11 +1302,18 @@ static int fill_mem_usage_elems(enum mem_zone zone, enum sof_ipc_dbg_mem_zone ip
 	return elem_number;
 }
 
+#if CONFIG_CORE_COUNT > 1
+#define PLATFORM_HEAP_SYSTEM_SHARED_CNT (PLATFORM_HEAP_SYSTEM_SHARED + PLATFORM_HEAP_RUNTIME_SHARED)
```

Comment:
maybe put this define into the same file with `PLATFORM_HEAP_SYSTEM_SHARED `

Reply:
heap counters are platform specific values, while this one is more generic (common for each multi-core platform), so keeping then in the same place doesn't sound like a good idea.